libcudacxx/include/cuda/std/detail/libcxx/include/__expected/expected_base.h

```diff
@@ -365,11 +365,13 @@ struct __expected_storage : __expected_destruct<_Tp, _Err>
 {
   using __base = __expected_destruct<_Tp, _Err>;
 
-#if defined(_LIBCUDACXX_COMPILER_NVRTC) // nvbug3961621
+// nvbug3961621
+#if defined(_LIBCUDACXX_COMPILER_NVRTC)  \
+ || (defined(_LIBCUDACXX_COMPILER_NVCC_BELOW_11_3) && defined(_LIBCUDACXX_COMPILER_CLANG))
```

Comment:
nit: should we update the comment on `#else` part?

Reply:
I feared someone would ask this

Reply:
Done